### PR TITLE
include api monitoring controller

### LINKED ISSUE #1921: Include api-monitoring-controller

### PR DESCRIPTION
**Pull request by maxim-tschumak**
Closes #1921

### CODE REVIEW

**Comment by maxim-tschumak** [comment]
I cannot see the `ci/jenkins/*` errors: `mtschumak is missing the Overall/Read permission`. Could you please have a look?

**Comment by mikkeloscar** [comment]
@maxim-tschumak gave you access to jenkins now and re-triggered the build.

**Comment by zeitlinger** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
The tests are stuck because it's enabled in the e2e environment, but no credentials are available there.

**Comment by maxim-tschumak** [comment]
@aermakov-zalando Do you have an idea how we can fix this?

**Comment by aermakov-zalando** [comment]
Add some conditionals based on `.Cluster.environment` in the manifests; e2e tests run with environment set to `e2e`. Or do you want to test the whole implementation?

**Comment by aermakov-zalando** [comment]
By the way, do the services used by the controller accept sandbox tokens?

**Comment by maxim-tschumak** [comment]
@aermakov-zalando thanks!

> Or do you want to test the whole implementation?

At this point not, but maybe eventually.

> By the way, do the services used by the controller accept sandbox tokens?

Yes. We are already thinking about how to implement this.

File: cluster/manifests/api-monitoring-controller/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+apiVersion: "extensions/v1beta1"
```
Comment (mikkeloscar) [inline cluster/manifests/api-monitoring-controller/deployment.yaml]: please use `apps/v1` here

**Comment by maxim-tschumak** [comment]
Please do not merge yet. We are currently discussing if this is the way we want to go. @tkrop @zeitlinger  will follow up on this.

**Comment by tkrop** [comment]
Substituted by #1956.

**Pull request closed without merging**